admin/attributes_controller.php

```diff
@@ -605,7 +605,7 @@
 function translate_type_to_name($opt_type)
 {
   global $products_options_types_list;
-  return $products_options_types_list[$opt_type];
+  return $products_options_types_list[$opt_type] ?? '';
```

Comment:
Perhaps, to drive home the fact that the 'name' is missing, use `TEXT_NONE` instead of an empty string?